containers/be-redis-pm2/Dockerfile

```
@@ -31,34 +31,40 @@ LABEL git="https://github.com/lehuygiang28/vphim"
 LABEL author="lehuygiang28 <lehuygiang28@gmail.com>"
 LABEL org.opencontainers.image.maintainer="lehuygiang28 <lehuygiang28@gmail.com>"
 
-# Install Redis and dependencies for cloudflared
+# Install Redis and Supervisor
 RUN apk update && \
```

Hadolint

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)